packages/components/src/components/tile/tile.e2e.ts

```diff
@@ -304,7 +304,27 @@ describe("calcite-tile", () => {
           {
             "--calcite-tile-accent-color-press": {
               shadowSelector: `.${CSS.container}`,
-              targetProp: "boxShadow",
+              targetProp: "outline",
+            },
+          },
+        );
+      });
+      describe(`applies to selected item's outline using selection-appearance="highlight"`, () => {
+        themed(
+          html`
+            <calcite-tile
+              heading="Tile heading lorem ipsum"
+              interactive
+              selection-mode="single"
+              selection-appearance="highlight"
+              selected
+            >
+            </calcite-tile>
+          `,
+          {
+            "--calcite-tile-accent-color-press": {
+              shadowSelector: `.${CSS.container}`,
+              targetProp: "outline",
             },
```

Comment:
If we're not going to add a public css token for developers to use for changing the highlight background color, we should add this additional token test that ensures the correct core token is being applied correctly:

```suggestion
            },
            "--calcite-color-surface-highlight": {
              shadowSelector: `.${CSS.container}`,
              targetProp: "backgroundColor",
            },
```

Reply:
You can use this instead with the new public token once you apply it in the CSS:

```suggestion
            },
            "--calcite-tile-background-color-highlight": {
                shadowSelector: `.${CSS.container}`,
                targetProp: "backgroundColor",
            }
```